tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -13181,6 +13181,12 @@ print_function_signature(int func_sptr, const char *fn_name, LL_ABI_Info *abi,
     /* Nobody sets -x 183 0x10, besides Flang. We're disabling LLVM inlining for
      * proprietary compilers. */
     print_token(" noinline");
+  } else if (XBIT(14, 0x8)) {
```

Comment:
Can we reuse XBIT(183, 0x10) for noinline? If not can the two else ifs be combined?

Reply:
The comment for `XBIT(183, 0x10)` suggests that it can be removed if nobody (e.g. from NVIDIA/PGI) cares about non-Flang user of flang2.

Reply:
I'll combine the two conditions. That okay?